docs/guides/admin/docs/workflowoperationhandlers/log-woh.md

```diff
@@ -0,0 +1,39 @@
+LoggingWorkflowOperationHandler
+===============================
+
+
+Description
+-----------
+
+The LoggingWorkflowOperationHandler is primarily meant for testing and debugging purposes. It allows to log the current
+state of of a workflow and/or its media package.
+
+|Name                |Default|Description                                                  |
+|--------------------|-------|-------------------------------------------------------------|
+|directory           |       |If set, write the logs to this directory                     |
+|workflowinstance-xml|`false`|Log the current state of the workflow as XML                 |
+|mediapackage-xml    |`false`|Log the state of the current workflow's media package as XML |
+|mediapackage-json   |`true` |Log the state of the current workflow's media package as JSON|
+
+If any format is specified, it will overwrite all defaults.
```

Comment:
What does it mean to specify format, here? The filename looks fixed by the code. 

It might be useful to note that setting multiple to true will output multiple, setting all to false will output nothing.

Reply:
Would this be better?

> Setting any output configuration (`*-xml` and `*-json` options) will overwrite all defaults and *only* the formats explicitly enabled will be logged.

Reply:
ping :-)

Reply:
Yes, I find that more clear. Thanks

Reply:
Sorry, I didn't realize I had set myself as reviewer! I'll finish this up.